egs2/msp_podcast/ser1/run.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train"
+valid_set="valid"
+test_sets="test test1 test2 valid"
+
+ser_config=conf/train_ser.yaml
+inference_config=conf/decode_asr.yaml
+local_data_opts="--lowercase false --remove_punctuation false --remove_emo X_O"
+# local_data_opts: 5 following options can be set (default=false)
+#--lowercase
+#   Convert transcripts into lowercase if "true".
+#--remove_punctuation
+#   Remove punctuation (except apostrophes) if "true".
+#--remove_tag
+#   Remove [TAGS] (e.g.[LAUGHTER]) if "true".
+#--remove_emo
+#   Remove the utterances with the specified emotional labels.
+#   If specifying two or more labels, concatenate them with "_" (e.g. xxx_exc_fru_fea_sur).
+#   emotional labels: ang (anger), hap (happiness), exc (excitement), sad (sadness),
+#   fru (frustration), fea (fear), sur (surprise), neu (neutral), and xxx (other)
+#--convert_to_sentiment
+#   Convert emotion to sentiment (Positive, Negative and Neutral)
+#   mapping from emotion to sentiment is as follows:
+#   Positive: hap, exc, sur
+#   Negative: ang, sad, fru, fea
+#   Neutral: neu
```

Comment:
Given the different definition for each dataset, I guess you might need to have an option to control the categories of emotion?